contributed_definitions/NXcontainer.nxdl.xml

```diff
@@ -151,12 +151,14 @@
 			adsorption of the container to be calculated.
 		</doc>
 	</group>
-	<group name="empty_measurement" type="NXentry">
+	<group name="reference_measurement" type="NXentry">
```

Comment:
Corrected as requested. I was confused by this bit of the docs:
http://download.nexusformat.org/doc/html/nxdl_desc.html#link
Which states: "A link element can only be a child of a field or group element"